tendrl/commons/message.py

```diff
@@ -153,7 +153,7 @@ def serialize_message(obj):
         serial = obj.isoformat()
         return serial
     elif isinstance(obj, Exception):
-        return yaml.safe_dump(obj)
+        return yaml.safe_dump(obj.message)
```

Comment:
@GowthamShanmugam I tried it and it seems to work but it gives unrelated python lingos which might not be desirable. See below
>>> from ruamel import yaml
>>> ex = Exception('Error!!')
>>> yaml.dump(ex)
'!!python/object/apply:exceptions.Exception [Error!!]\n'